homeassistant/components/abode/__init__.py

```diff
@@ -158,9 +158,9 @@ def capture_image(call):
             signal = f"abode_camera_capture_{entity_id}"
             dispatcher_send(hass, signal)
 
-    def trigger_automation(call):
+    def trigger_automation(call: ServiceCall) -> None:
         """Trigger an Abode automation."""
-        entity_ids = call.data.get(ATTR_ENTITY_ID)
+        entity_ids = call.data[ATTR_ENTITY_ID]
```

Comment:
The attributes are required by the schema:
https://github.com/home-assistant/core/blob/00a4b60f741967c6d93d2c49c2de5490264a356c/homeassistant/components/abode/__init__.py#L55